operator/Makefile

```diff
@@ -9,14 +9,14 @@ TARGETS := cilium-operator cilium-operator-generic cilium-operator-aws cilium-op
 
 all: $(TARGETS)
 
-cilium-operator: OPERATOR_TAGS=ipam_provider_aws,ipam_provider_azure,ipam_provider_operator
-cilium-operator-generic: OPERATOR_TAGS=ipam_provider_operator
-cilium-operator-aws: OPERATOR_TAGS=ipam_provider_aws
-cilium-operator-azure: OPERATOR_TAGS=ipam_provider_azure
+cilium-operator: GO_TAGS_FLAGS+=ipam_provider_aws,ipam_provider_azure,ipam_provider_operator
```

Comment:
:exploding_head: :100: 

I didn't know that this is possible.

Reply:
I believe it is specific to GNU Make

Reply:
The magic of target-specific variable values in GNU make :sparkles: :smile: https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html

Reply:
thanks both for info, I can simplify some of the stuffs I have at work with this :100: